support/entry/entry.cpp

```diff
@@ -361,13 +375,15 @@ int main(int argc, const char** argv) {
       }
     }
     std::thread main_thread([&entry_data, &return_value]() {
-      while(!entry::k_stream_started) {} // Wait for the stream to be started
+      while (!entry::k_stream_started) {
+      }  // Wait for the stream to be started
       return_value = main_entry(&entry_data);
     });
     std::atomic<bool> exited(false);
     std::thread ggp_thread([&exited]() {
-      while(!exited) {
-        while(ggp_event_queue->ProcessEvent()) {}
+      while (!exited) {
+        while (ggp_event_queue->ProcessEvent()) {
```

Comment:
really minor nit: this style change appears different from how empty functions are handled (based on a change above) - not sure if intentional or not so just figured I'd mention it.

Reply:
I defer to clang-format for all my formatting needs!